site/earlgrey-diagram/src/metadata.js

```diff
@@ -0,0 +1,239 @@
+export default {
+    "opentitan-logo": {
+        title: "opentitan",
+        href: "/",
+    },
+    "high-speed-crossbar": {
+        title: "high-speed-crossbar",
+        metrics: "tlul",
+        href: "/hw/ip/tlul#top",
+        report: "/hw/top_earlgrey/ip/xbar_main/dv/autogen",
+    },
+    "ibex": {
+        title: "ibex",
```

Comment:
`title` is visible to users, right? If so, I think we should uniformly use the value of the `human_name` field (which many but not all HW blocks already have; [example](https://github.com/lowRISC/opentitan/blob/6609ad5a645dc3480bc68d3ff80b2c46be3f7138/hw/ip/aes/data/aes.hjson#L8)) for it.

Reply:
The `title` field is used within the hover, see the image below.
I think standardizing on a single source of truth for data (ie the .hjson) is desirable here too, but currently not all blocks have suitable data. As you pointed out, some don't have `human_name` at all, and some have a 'human_name' which is probably too long for the intended location. (e.g. rv_core_ibex.hjson -> `human_name: RISC-V Core Wrapper for Ibex`)

How about using the names as they are in the PR, and creating a follow-up issue to ensure all IP blocks have suitable .hjson metadata, then switch over to using that instead?

![Screenshot from 2023-03-10 14-09-18](https://user-images.githubusercontent.com/102029880/224337451-93160cd4-ca3f-4965-b12a-34c0d4027c48.png)

Reply:
SGTM, thanks

Reply:
How does that translate to things like "SPI Host 1", where they are numbered?
Does the `human_name` have a number there? do we even want the number in the tooltip?

Reply:
I would guess we don't want an instantiation number in the tooltip, because there aren't a separate set of tests per instantiation but having the number may give this impression.

Reply:
Please create an issue to track this.